test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
```

Comment:
While it might not seem obvious, we have a sort order for our dependencies:

1. Core library (e.g. `assert`, `fs`)
2. Third party libraries (e.g. `request`, `grunt`)
3. First party libraries (e.g. `../package.json`, `./utils/config`)

```js
var assert = require('assert');
var fs = require('fs');
var configUtils = require('./utils/config');
var testUtils = require('./utils/test');
var pkg = require('../package.json');
```